src/nrnpython/nrnpython.cpp

```diff
@@ -45,16 +50,26 @@ int nrnpy_site_problem;
 
 void nrnpy_augment_path() {
   static int augmented = 0;
-  char buf[1024];
   if (!augmented && strlen(neuronhome_forward()) > 0) {
-    // printf("augment_path\n");
     augmented = 1;
-    sprintf(buf, "sys.path.append('%s/lib/python')", neuronhome_forward());
     int err = PyRun_SimpleString("import sys");
     assert(err == 0);
-    err = PyRun_SimpleString(buf);
-    assert(err == 0);
-    sprintf(buf, "sys.path.prepend('')");
+#if defined(__linux__) || defined(DARWIN)
+    // If /where/installed/lib/python/neuron exists, then append to sys.path
+    std::string lib = std::string(path_prefix_to_libnrniv());
+    // For an autotools build on an x86_64, it ends with x86_64/lib/
+    size_t pos = lib.find(NRNHOSTCPU "/lib/", lib.length() - (5+strlen(NRNHOSTCPU)));
```

Comment:
another minor comment : is `5` refers in strlen of "/lib/" ? (may be we can replace to be more clear)

Reply:
I was thinking of the very strange case of a ```CMAKE_INSTALL_PREFIX=/home/hines/foo/x86_64/lib/neuron/nrn/build/install```
The nrnpython.cpp file has no notion of whether the build was autotools or cmake

Reply:
I'm  going to use the NRNCMAKE define so that that code fragment is executed only for an autotools build because it
occurred to me tha lib could have a length less than 11

Reply:
Ok. I don't know the details but anything robust would be nice :)